tests/callout.c

```diff
@@ -17,7 +18,7 @@ int test_callout() {
     callout_process(i);
   }
 
-  return 0;
+  return KTEST_SUCCESS;
 }
 
-KTEST_ADD(callout, test_callout);
+KTEST_ADD_FLAGS(callout, test_callout, KTEST_FLAG_BROKEN);
```

Comment:
Running this test causes a crash the next time a clock interrupt arrives. I'd rather not fix this on my own as I'm not familiar with callouts, so I've excluded this test from running in `test=all` mode by marking it as `BROKEN`.